scripts/generate-go-crd-clients/client_test.go

```diff
@@ -151,9 +152,9 @@ func TestComputeInstanceGoClient(t *testing.T) {
 					SubnetworkRef: &v1alpha1.ResourceRef{
 						External: "default",
 					},
-					NetworkIpRef: &v1alpha1.ResourceRef{
-						Kind: "ComputeAddress",
-						Name: computeAddressName,
+					NetworkIpRef: &computev1beta1.InstanceNetworkIpRef{
+						Kind: &networkIpKind,
+						Name: &computeAddressName,
```

Comment:
according to the schema in https://cloud.google.com/config-connector/docs/reference/resource-docs/compute/computeinstance

* NetworkIpRef should have these fields (all strings are optional)
  ```
  networkIpRef:
    external: string
    kind: string
    name: string
    namespace: string
   ```
* SubnetworkRef should have these fields (all strings are optional)
  ```
  subnetworkRef:
    external: string
    name: string
    namespace: string
  ```

so it's a bit odd that:
* SubnetworkRef is a ResourceRef which has a Kind field that isn't supported in the schema
* Fields are `string`s in ResourceRef, and `*string`s in InstanceNetworkIpRef. I'm not sure which is preferable but it'd be good to be consistent.